db/seeders/letter_template_seeder.rb

```diff
@@ -57,17 +63,135 @@ def seed!
+                  <br>Enclosed is all the information related to your request that I am able to release. Some information may have been withheld and this is because the information is exempt from disclosure under the DPA. The exemptions within the DPA include information which is processed for the prevention or detection of a crime or the apprehension or prosecution of offenders, and information that would identify third parties. Where we have withheld exempt information, you will see items redacted on the documents.
+                  <br>
+                  <br>I can confirm that the personal data contained within these documents is being processed by the Ministry of Justice for the purposes of the administration of justice and for the exercise of any functions of the Crown, a Minister of the Crown or a government department. As such we may share or exchange data with other Departments or organisations if it is lawful to do so, for example the Police or the Probation Service.
+                  <br>
+                  <br>If you have any queries regarding your request please contact the Offender Subject Access Request Team, at the address above. It is also open to you to ask the Information Commissioner to look into the case. You can contact the Information Commissioner at this address:
+                  <br>
+                  <br>Information Commissioner's Office, Wycliffe House, Water Lane, Wilmslow, Cheshire, SK9 5AF
+                  <br>Internet: ico.org.uk
+                  <br>
+                  <br>Please note that copies of the data provided to you will be retained for no longer than nine months. Once this period has passed, we will be unable to answer any questions you may have or provide duplicates of this information. It will not normally be disclosed in any future SARs.
+                  <br>
+                  <br>Finally, I would like to suggest that you do not keep this information where it can be accessed by others. Once you have read through the information it can be placed in your stored property.
+                  <br>
+                  <br>Yours sincerely
+                  <br>
+                  <br>
+                  <br>
+                  <br>
+                  <br>Despatch Team
+                  <br>Offender Subject Access Request Team
+                  <br>Ministry of Justice
+                  </p>
+                EOF
+                )
+
+    rec = LetterTemplate.find_by(abbreviation: 'prisoner-disclosed-covid')
+    rec = LetterTemplate.new if rec.nil?
+    rec.update!(name: 'Prisoner disclosed letter (COVID-19)',
+                abbreviation: 'prisoner-disclosed-covid',
+                template_type: 'dispatch',
+                body: <<~EOF
+                  <p><%= values.recipient_name %><% if values.prison_number.present? %> - <%= values.prison_number %><% if (values.prison_number.gsub(/[,]/, ' ').squeeze(' ').strip).match?(/[ ]/) %> [DELETE AS APPROPRIATE]<% end %><% end %>
```

Comment:
Maybe at some point it would be nice to get these into a method maybe on the decorator or in the letter class

```suggestion
                  <p><%= values.recipient_name %><% if values.prison_number.present? %> - <%= values.prison_number %><% if (values.prison_number.gsub(/[,]/, ' ').squeeze(' ').strip).match?(/[ ]/) %> [DELETE AS APPROPRIATE]<% end %><% end %>
```

Reply:
Happy to circle round and extract this into a method or decorator later.